main.rb

```diff
@@ -1,7 +1,9 @@
 require_relative './app'
+require './data'
+require 'json'
 
 puts 'Welcome to your school library!!!'
-
+initialize_files
 def main
   app = App.new
   app.start
```

Comment:
- [ ] I getting the error below while running your project. Kindly fix it.
![Screenshot from 2022-05-26 14-44-28](https://user-images.githubusercontent.com/65860678/170490854-525091b7-ec85-4325-ae89-7c16099b7d0d.png)